lib/qesapdeployment.pm

```diff
@@ -87,6 +91,56 @@ sub qesap_create_folder_tree {
     assert_script_run("mkdir -p $paths{deployment_dir}", quiet => 1);
 }
 
+=head3 qesap_get_variables
+
+    Scans yaml config for '%OPENQA_VARIABLE%' placeholders and searches for values in OpenQA defined variables.
+    Returns hash with openqa variable key/value pairs.
+=cut
+
+sub qesap_get_variables {
+    my %paths = qesap_get_file_paths();
+    my $yaml_file = $paths{'qesap_conf_src'};
+    my %variables;
+    my $grep_cmd = "grep -v '#' | grep -oE %[A-Z0-9_]*% | sed s/%//g";
+    my $cmd = join(" ", "curl -s -L", $yaml_file, "|", $grep_cmd);
```

Comment:
Please change to **curl -s -fL** by adding the **--fail** argument. It results in culr to clearly fails if the $yaml_file is not there. We recently applied the same fix in all other curl cmd on this lib file

Reply:
Fixed